tchannel/tornado/tchannel.py

```diff
@@ -221,7 +228,7 @@ def request(self,
                                   retry=retry,
                                   **kwargs)
 
-    def listen(self, port=None):
+    def listen(self, port=None, backlog=2048):
```

Comment:
Use `None` for default in parameter list and set default value inside the body. This allows you to not duplicate the default value in all the callers (since they can just pass None).